### PR TITLE
apache-tinkerpop-gremlin(-console)(-server)@3.6.0: Fix bin

### PR DESCRIPTION
**Pull request by ciaranchen**
remove bin from apache-tinkerpop-gremlin-*

Relates to #8842 

- [*] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/apache-tinkerpop-gremlin-console.json [hunk 1]
```diff
@@ -6,7 +6,6 @@
     "url": "https://www.apache.org/dist/tinkerpop/3.6.0/apache-tinkerpop-gremlin-console-3.6.0-bin.zip",
     "hash": "sha512:727061ee2874087bc889cd26aa651537c96eaa84d9eaed568902de67d0765a9acca4cb6efa11786a45e6fdde5242df908ab2d84ea925dd87ae907537949b108a",
     "extract_dir": "apache-tinkerpop-gremlin-console-3.6.0",
```
Comment (issaclin32) [inline bucket/apache-tinkerpop-gremlin-console.json:8]: You don't have to remove **bin**. There is a better way to solve this.

```suggestion
    "extract_dir": "apache-tinkerpop-gremlin-console-3.6.0",
    "pre_install": [
        "$cont = @(",
        "    '@echo off'",
        "    'cd /d \"%~dp0\"'",
        "    'gremlin.bat'",
        ")",
        "Set-Content \"$dir\\bin\\gremlin-scoop.bat\" $cont -Encoding ascii"
    ],
    "bin": [
        [
            "bin\\gremlin-scoop.bat",
            "gremlin"
        ]
    ],
```

File: bucket/apache-tinkerpop-gremlin-server.json [hunk 2]
```diff
@@ -6,7 +6,6 @@
     "url": "https://www.apache.org/dist/tinkerpop/3.6.0/apache-tinkerpop-gremlin-server-3.6.0-bin.zip",
     "hash": "sha512:2980c3e5c85cf7b6944b250b39a5e2e3423f476cd828dcd756d0ef59b78d7714854df769581b936409c59b9e4deb1a19cecfff3a69534919f7ecd53467d1e543",
     "extract_dir": "apache-tinkerpop-gremlin-server-3.6.0",
```
Comment (issaclin32) [inline bucket/apache-tinkerpop-gremlin-server.json:8]:
```suggestion
    "extract_dir": "apache-tinkerpop-gremlin-server-3.6.0",
    "pre_install": [
        "$cont = @(",
        "    '@echo off'",
        "    'cd /d \"%~dp0\"'",
        "    'gremlin-server.bat'",
        ")",
        "Set-Content \"$dir\\bin\\gremlin-server-scoop.bat\" $cont -Encoding ascii"
    ],
    "bin": [
        [
            "bin\\gremlin-server-scoop.bat",
            "gremlin-server"
        ]
    ],
```

**Comment by ciaranchen** [comment]
Thank you for your suggestions. I plan to open another pr and fix this in a single commit.

**Comment by issaclin32** [comment]
superseded by #8855

**Pull request closed without merging**